src/js/utils/form-values-utils.jsx

```diff
@@ -52,3 +52,12 @@ export const getReceivingPayloadContainers = formValues =>
       return _.omit(item, 'product.displayNames');
     }),
   }));
+
+/** Expects to have filterValue already in lower case */
+export const matchesProductCodeOrName = ({
+  productCode, productName, displayName, filterValue,
+}) =>
+  (productCode?.toLowerCase()?.includes(filterValue) ||
+    productName?.toLowerCase()?.includes(filterValue) ||
+      displayName?.toLowerCase()?.includes(filterValue)
+  );
```

Comment:
Do you think that maybe we could just pass a single **product** object and extract those values inside of this function ?

```suggestion
export const matchesProductCodeOrName = ({ product, filterValue }) => {
const productCode = product?.productCode;
const productName = product?.name;
const displayName = product?.displayName ?? product?.displayName?.default;

return (productCode?.toLowerCase()?.includes(filterValue) ||
    productName?.toLowerCase()?.includes(filterValue) ||
    displayName?.toLowerCase()?.includes(filterValue)
  );
}  
```

Reply:
sure I can do it, I just used the fact, that I had access directly to the `productCode` and `productName` from the DTO, not having to access it via `product.productCode`, but it doesn't matter and it's cleaner doing it the way you suggested. Thanks